analysis/utilityFunctions.py

```diff
@@ -245,3 +246,17 @@ def mergeRawData(rootpath, substr, output_file, remove_raw=False, sep_str='\t\t'
 	for i in range(num_rows):
 		fout.write(all_data[i] + '\n')
 	fout.close()
+
+######################################
+# event_data_obj
+# This is a helper class for the stc_analysis_class 

```

Comment:
`stc_analysis_class` is now `stc_data_obj`, right?

Reply:
Fixed this.